contrib/actions/cds-nexus-upload.hcl

```diff
@@ -0,0 +1,63 @@
+
+name = "CDS_NexusUpload"
+description = "Upload file on Nexus"
+
+// Requirements
+requirements = {
+	"bash" = {
+		type = "binary"
+		value = "bash"
+	}
+	"login" = {
+		type = "string"
```

Comment:
set a default value for login parameter (which should be in parameters section not in requirements) = {{.cds.proj.nexus.login}}